src/components/common/header/PageHeader.tsx

```diff
@@ -33,40 +40,137 @@ function PageHeader({
     showCloseButton = false,
     onClose,
 }: PageHeaderType) {
+    const [showHelpCard, setShowHelpCard] = useState(false)
+    const [showLogOutCard, setShowLogOutCard] = useState(false)
+    const [loginInfo, setLoginInfo] = useState(undefined)
+    const email: string = loginInfo ? loginInfo['email'] || loginInfo['sub'] : ''
+    const [currentServerInfo, setCurrentServerInfo] = useState<{ serverInfo: ServerInfo; fetchingServerInfo: boolean }>(
+        {
+            serverInfo: undefined,
+            fetchingServerInfo: false,
+        },
+    )
+
+    const getCurrentServerInfo = async () => {
+        try {
+            const { result } = await getServerInfo()
+            setCurrentServerInfo({
+                serverInfo: result,
+                fetchingServerInfo: false,
+            })
+        } catch (err) {
+            setCurrentServerInfo({
+                serverInfo: currentServerInfo.serverInfo,
+                fetchingServerInfo: false,
+            })
+            console.error('Error in fetching server info')
+        }
+    }
+    useEffect(() => {
+        getCurrentServerInfo()
+    }, [])
+
+    useEffect(() => {
+        setLoginInfo(getLoginInfo())
```

Comment:
Remove this useEffect. 
getLoginInfo() is returning data from cookies so no need to maintain an extra state. It can be kept in some var at the top.

Reply:
Done